client/js/lounge.js

```diff
@@ -193,7 +195,9 @@ $(function() {
 		);
 		renderChannel(data.chan);
 		var chan = sidebar.find(".chan")
-			.sort(function(a, b) { return $(a).data("id") - $(b).data("id"); })
+			.sort(function(a, b) {
+				return $(a).data("id") - $(b).data("id");
+			})
```

Comment:
FWIW, [4.x mostly supports arrow functions.](http://node.green/#arrow-functions) (there are a few edge cases it doesn't support, but it's unlikely to be something we ever have to deal with)